lint-config.edn

```diff
@@ -0,0 +1,9 @@
+^:replace
```

Comment:
I adopted the config filename and content as mentioned in #14808, feel free to suggest any improvements.